templates/web/base/admin/contact-form.html

```diff
@@ -0,0 +1,137 @@
+<form method="post" action="[% c.uri_for('body', body_id ) %]" enctype="application/x-www-form-urlencoded" accept-charset="utf-8" id="category_edit">
+
+    <div class="admin-hint">
+      <p>
+        [% loc('Choose a <strong>category</strong> name that makes sense to the public (e.g., "Pothole", "Street lighting") but is helpful
+                to the body too. These will appear in the drop-down menu on the report-a-problem page.') %]
+        <br>
+        [% loc("If two or more bodies serve the same location, FixMyStreet combines identical categories into a single entry in
+                the menu. Make sure you use the same category name in the bodies if you want this to happen.") %]
+      </p>
+    </div>
+    <p>
+      <strong>[% loc('Category') %] </strong><input type="text" class="form-control" name="category" size="30" value="[% contact.category | html %]" required>
```

Comment:
The category isn't editable on the edit page; it certainly would be nice to, somehow (issues with problem table notwithstanding), but I think here it will just then create a new contact with the new data given.